File: js/1828.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')


n = parseInt(lines.shift())

for (i=0;i<n;i++){
  e = lines.shift().split(' ')

  sheldon = e[0].trim()
  raj = e[1].trim()

  if (sheldon === raj) {
    console.log(`Caso #${i + 1}: De novo!`);
} else if (sheldon === "tesoura" && raj === "papel") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "tesoura" && raj === "lagarto") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "papel" && raj === "pedra") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "papel" && raj === "Spock") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "pedra" && raj === "tesoura") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "pedra" && raj === "lagarto") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "Spock" && raj === "pedra") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "Spock" && raj === "tesoura") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "lagarto" && raj === "Spock") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else if (sheldon === "lagarto" && raj === "papel") {
    console.log(`Caso #${i + 1}: Bazinga!`);
} else {
    console.log(`Caso #${i + 1}: Raj trapaceou!`);
}
}

File: python/1828.py
n = int(input())


for i in range(n):
    e = input().split()
    sheldon = e[0]
    raj = e[1]
    if sheldon == raj:
        print(f"Caso #{i + 1}: De novo!")
    elif sheldon == "tesoura" and raj == "papel":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "tesoura" and raj == "lagarto":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "papel" and raj == "pedra":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "papel" and raj == "Spock":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "pedra" and raj == "tesoura":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "pedra" and raj == "lagarto":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "Spock" and raj == "pedra":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "Spock" and raj == "tesoura":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "lagarto" and raj == "Spock":
        print(f"Caso #{i + 1}: Bazinga!")
    elif sheldon == "lagarto" and raj == "papel":
        print(f"Caso #{i + 1}: Bazinga!")
    else:
        print(f"Caso #{i + 1}: Raj trapaceou!")
